src/test/java/org/sagebionetworks/bridge/exporter/integration/Exporter3Test.java

```diff
@@ -1,5 +1,11 @@
 package org.sagebionetworks.bridge.exporter.integration;
 
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.fluent.Request;
+import org.apache.http.util.EntityUtils;
```

Comment:
We put regular imports after static imports. Is your code style config not updated? Note that the code style configs are set independently on each project, so you may need to import them again.